sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -562,37 +642,51 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
         "license" -> JSON.str("UNLICENSED")
       ),
 
-      npmUpdate := NpmUpdateTasks.npmUpdate(
-      baseDirectory.value,
-      (crossTarget in npmUpdate).value,
-      scalaJSBundlerPackageJson.value.file,
-      useYarn.value,
-      scalaJSNativeLibraries.value.data,
-      streams.value,
-      npmExtraArgs.value,
-      yarnExtraArgs.value),
+      npmUpdate := npmInstall.value,
+
+      crossTarget in npmInstall := (crossTarget in npmUpdate).value,
+
+      npmInstall := {
+        val dir = npmInstallDependencies.value
+        val _ = npmInstallJSResources.value
+        dir
+      },
```

Comment:
Do we need `npmInstallDependencies` to be executed *before* `npmInstallJSResources`? There is no guarantee that this will be the case. On the other hand, if we don’t need this guarantee, then we can implement `npmInstall` as follows:

~~~ scala
npmInstall := {
  val _ = npmInstallJSResources.value
  npmInstallDependencies.value
}
~~~

Reply:
I guess there might be a race condition if both tasks try to create the target directory at the same time?